vowpalwabbit/gd_mf.cc

```diff
@@ -264,7 +264,7 @@ void save_load(gdmf& d, io_buf& model_file, bool read, bool text)
 
   if (model_file.num_files() > 0)
   {
-    if (!all.weights.not_null()) { THROW("Error: Model weights not initialized."); }
+    if (!all.weights.not_null()) { THROW("Model weights not initialized when trying to load into them."); }
```

Comment:
I'm not sure I understand the `when trying to load into them`, is it `when trying to load them`?

Reply:
This essentially means the weights object was not initialized/created/allocated before we try to write to it